### DISCUSSION TITLE
Select options depending of the previous group?

### DISCUSSION THREAD

**Discussion by tetienne** [charmbracelet/huh#72] [Q&A]
Many commands in my CLI tool require user input, particularly for AWS-related operations. Typically, the process involves selecting an ECS cluster first, followed by a service, and so on. Each subsequent step is contingent on the choices made in the previous one.

In the examples I've seen, the options available in a huh form's select field are fixed. Is there a way to make select options dynamic?

**Comment by meowgorithm** [comment]
Hi! See #75 for an example on how to do this.

In the future we may support something more official in the API.
